Assets/MRTK/Providers/Oculus/XRSDK/MRTK-Quest/Audio/README.md

```diff
@@ -0,0 +1,2 @@
+[MRTK-Audio](https://github.com/joncohenproducer/MRTK-Audio) is a collection of open source audio assets taken from [Jon Cohen](https://twitter.com/joncohensound)'s github, to add some flavor to teleport interactions, and background music.
```

Comment:
Can we do this without adding another third party dependency? It's great if people can optionally pull that repo and assign sounds, but I don't think we should pull it into this repo (especially since it comes with its own, non-MIT license).

Reply:
I'd very much like to see the audio remain. I had a chat with @wiwei about the license, and it seemed like Apache 2 is ok to be ingested. The only difference it has from MIT is that the contents can't be copyrighted by people.

Reply:
Ingesting repos like this can lead to GUID clashes (easily fixable, yeah) and getting out of sync with the source. It'd be good to include a guide or pointer to using both of these together in an app, but I feel like adopting the license and importing the files are decisions best left to devs, plus they'd be getting the latest and greatest audio clips instead of whenever we decided to snap.

We also already have a set of audio clips: https://github.com/microsoft/MixedRealityToolkit-Unity/tree/mrtk_development/Assets/MRTK/StandardAssets/Audio 

Hover Click.wav sounds somewhat similar (at least in intent) to several of the existing clips (ButtonPress, or Manipulation_Start for a louder version, or maybe Shell_Click_In or Select_Secondary).
New Object Interaction.wav does sound a bit different than any of our clips, but we have existing "interaction" noises throughout that could be the default.

I feel like providing a consistent set of audio clips, with guides on other sets to use (like the MRTK-Audio repo) would be better than pulling these in ourselves. Adding @cre8ivepark if we feel like certain sounds are missing from the default set.

Reply:
Fair enough for the HoverClick sound, though I will say the current clips are very faint. I was considering redoing all the audio with the friend who made these new clips, but the audio workflow is currently very lacking, as all clips are serialized references on a per-prefab basis, which makes migrating audio in bulk very cumbersome.

Back to the point though - I think HoverClick is tonally similar enough to the existing clips that it's not problematic to use alongside existing sounds. Having this sound be a fair bit louder helps it stick out as well, but that's a subjective point. New Object Interaction sounds a bit different than other sounds, though I firmly believe no existing sounds appropriately convey the sense of displacement that it does, as related to the Teleport Confirmation sound.

The rest of the sounds, besides "Background Menu Music" aren't used in MRTK-Quest, and you may feel free to remove them if you find that they're not tonally appropriate to sit alongside existing MRTK sounds. 

I think having some basic music loop that fits with the existing soundscape tonally could be appealing as well.

Reply:
Hey Eric, we're talking to some other folks on our team for making some in house versions of the audio to reduce the organizational+license jumble of dependencies, so even if not for this PR, we'll definitely be updating our soundscape in line with the improvements you've made!